docs/es/docs/tutorial/body.md

```diff
@@ -0,0 +1,213 @@
+    {!> ../../../docs_src/body/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.8+"
+
+    ```Python hl_lines="7-11"
+    {!> ../../../docs_src/body/tutorial001.py!}
+    ```
+
+Lo mismo que cuando se declaran parámetros de consulta, cuando un atributo del modelo tiene un valor predeterminado, no es obligatorio. En caso contrario, es obligatorio. Utiliza `None` para que sea opcional.
+
+Por ejemplo, este modelo declara un "`objeto`" JSON (o un `dict` de Python) como:
+
+```JSON
+{
+    "name": "Foo",
+    "description": "An optional description",
+    "price": 45.2,
+    "tax": 3.5
+}
+```
+
+...como `description` y `tax` son opcionales (con un `None` como valor por defecto), este "`objeto`" JSON "`object`" también sería válido:
+
+```JSON
+{
+    "name": "Foo",
+    "price": 45.2
+}
+```
+
+## Decláralo como parámetro
+
+Para agregarlo a su *operación de ruta*, decláralo de la misma manera que declaró los parámetros de la ruta y de la consulta:
+
+=== "Python 3.10+"
+
+    ```Python hl_lines="16"
+    {!> ../../../docs_src/body/tutorial001_py310.py!}
+    ```
+
+=== "Python 3.8+"
+
+    ```Python hl_lines="18"
+    {!> ../../../docs_src/body/tutorial001.py!}
+    ```
+
+...y declara su tipo como el modelo que creaste, `Item`.
+
+## Resultados
+
+Con solo esa declaración de tipo Python, **FastAPI** hará lo siguiente:
+
+* Lee el cuerpo de la <abbr title="tambien conocido como solicitud. En inglés: request">petición</abbr> como JSON.
+* Convierte los tipos correspondientes (si es necesario).
+* Valida los datos.
+     * Si los datos no son válidos, devolverá un error bonito y claro, indicando exactamente dónde y cuál fue el dato incorrecto.
+* Te dará los datos recibidos en el parámetro `item`.
+     * Como lo declaraste en la función como de tipo `Item`, también tendrás todo el soporte del editor (completado, etc.) para todos los atributos y sus tipos.
+* Genera definiciones del <a href="https://json-schema.org" class="external-link" target="_blank"><abbr title="tambien conocido en inglés como: JSON Schema">Esquema JSON</abbr></a> para su modelo, que también podrás usarlas en cualquier otro lugar que desees si tiene sentido para tu proyecto.
+* Esos esquemas formarán parte del esquema OpenAPI generado y serán utilizados por la documentación automática de las <abbr title="User Interfaces">UIs</abbr>.
+
+## Documentación automática
+
+Los <abbr title="tambien conocido en inglés como: JSON Schemas">Esquemas JSON</abbr> de tus modelos serán parte de su esquema generado por OpenAPI y se mostrarán en las documentaciones interactivas de la API.:
+
+<img src="/img/tutorial/body/image01.png">
+
+Y también se usará en la documentación de la API dentro de cada *operación de ruta* que los necesite:
+
+<img src="/img/tutorial/body/image02.png">
+
+## Soporte del editor
+
+En tu editor, dentro de tu función obtendrás sugerencias de tipo y completamiento en todas partes (esto no sucedería si recibiera un `dict` en lugar de un modelo Pydantic):
```

Comment:
What do you thing about using **auto-completado** instead of **completamiento**? 

```suggestion
En tu editor, dentro de tu función obtendrás sugerencias de tipo y autocompletado en todas partes (esto no sucedería si recibiera un `dict` en lugar de un modelo Pydantic):
```